src/com/studentmanagementsystem/logic/Student.java

```diff
@@ -0,0 +1,40 @@
+package com.studentmanagementsystem.logic;
+import com.studentmanagementsystem.data.list.ArrayListStudentDAO;
+
+public class Student {
+    private String studentFirstName;
```

Comment:
The prefix "student" is redundant since the first name is inside a "Student" class anyways, so just use "firstName" instead